automerge/src/automerge.rs

```diff
@@ -236,6 +236,11 @@ impl Automerge {
         }
     }
 
+    pub fn object_type<O: AsRef<ExId>>(&self, obj: O) -> Result<ObjType, AutomergeError> {
```

Comment:
Why not return an option since the error is just `Fail` anyway and the opset function returns an option too?